src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -81,11 +85,14 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
       connectionPoolConfig.getInt("pool-timeout-millis")
     )
 
+  val reconnectMaxDelay: FiniteDuration = config.getDuration("reconnect-max-delay", TimeUnit.MILLISECONDS).millis
+
   def clusterBuilder(clusterId: String)(implicit ec: ExecutionContext): Future[Cluster.Builder] = {
     lookupContactPoints(clusterId).map { cp =>
       val b = Cluster.builder
         .addContactPointsWithPorts(cp.asJava)
         .withPoolingOptions(poolingOptions)
+        .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
```

Comment:
no, it's the start delay, in milliseconds. so it's exponential between start and max.